src/icons/Contentful.js

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react'
+import PropTypes from 'prop-types'
+function SvgContentful({ size, color, style }) {
```

Comment:
Same for this file and line breaks between imports, function, prop types, and export.